lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -409,7 +409,7 @@ export class Vulnerabilities {
       this.isVersionGt(version, depVersion, versioningApi),
     );
     if (fixedVersion) {
-      return ecosystem === 'PyPI' ? `==${fixedVersion}` : fixedVersion;
+      return ecosystem === 'PyPI' ? `>=${fixedVersion}` : fixedVersion;
```

Comment:
Sure we can't fix all ecosystems at once?

Reply:
Sure, we can 👍